truffleruby/src/main/ruby/post-boot/post-boot.rb

```diff
@@ -38,6 +38,8 @@
       end
     end
 
+    require 'truffle/patching'
```

Comment:
Most of the rubygem files is loaded lazily, so it works. If we need to patch any of the eagerly loaded files it can be done manually here. I rather expect we'll be removing the patches rather than adding new ones, it should not be a problem.